crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -1628,7 +1643,9 @@ impl PgManager {
                         package: SuiAddress::from_array(**e.package_id),
                         name: e.transaction_module.to_string(),
                     }),
-                    event_type: Some(MoveType::new(e.type_.to_string())),
+                    event_type: Some(MoveType::new(
+                        e.type_.to_canonical_string(/* with_prefix */ true),
```

Comment:
Don't have a strong opinion here, but thoughts on controlling how type gets rendered in MoveType itself instead of when we pass the type string to MoveType?

Reply:
I guess that's beyond the scope of this PR which is just to correct usage of to_string though